### DISCUSSION TITLE
Analogue of Closure Tables

### DISCUSSION THREAD

**Discussion by euberdeveloper** [prisma/prisma#5571] [Q&A]
Hello, I have a doubt regarding self relations.

How does Prisma relate to these Typeorm functionalities https://typeorm.io/#/tree-entities/? Are all implementable?

**Comment by cgarfio** [comment]
I've found a really good slide [Models for hierachical data](https://www.slideshare.net/billkarwin/models-for-hierarchical-data), this can help. 

Also here are the docs for [Self Relation](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/self-relations) in prisma.io.

Jaskier#2888 (Discord) reach me out if you need some help, I'm currently working on a file system tree implementation.
